modular_zubbers/modules/modular_items/code/idmaco_donator.dm

```diff
@@ -8,7 +8,7 @@
 /datum/loadout_item/suit/idmarsuit
 	name = "IDMA service jacket"
 	item_path = /obj/item/clothing/suit/hooded/wintercoat/security/idma_jacket
-	ckeywhitelist = list("especiallystrange",  "snaffle15")
+	ckeywhitelist = list("especiallystrange", "waterpig",  "snaffle15")
```

Comment:
You should probably make a define- `#define IDMA_CKEY_WHITELIST list("[...`
That way you'd only have to edit it in one place

Reply:
Thanks <3